src/displayapp/screens/settings/SettingSetDate.cpp

```diff
@@ -0,0 +1,272 @@
+  lblMonth = lv_label_create(lv_scr_act(), NULL);
+  //lv_obj_set_style_local_text_font(lblMonth, LV_LABEL_PART_MAIN, LV_STATE_DEFAULT, &jetbrains_mono_42);
+  UpdateMonthLabel();
+
+  yearValue = static_cast<int>(dateTimeController.Year());
+  if (yearValue < 2021)
+    yearValue = 2021;
+  lblYear = lv_label_create(lv_scr_act(), NULL);
+  //lv_obj_set_style_local_text_font(lblYear, LV_LABEL_PART_MAIN, LV_STATE_DEFAULT, &jetbrains_mono_42);
+  lv_label_set_text_fmt(lblYear, "%d", yearValue);
+  lv_label_set_align(lblYear, LV_LABEL_ALIGN_CENTER);
+  lv_obj_align(lblYear, lv_scr_act(), LV_ALIGN_CENTER, POS_X_YEAR, POS_Y_TEXT);
+
+  btnDayPlus = lv_btn_create(lv_scr_act(), NULL);
+  btnDayPlus->user_data = this;
+  lv_obj_set_size(btnDayPlus, 50, 40);
+  lv_obj_align(btnDayPlus, lv_scr_act(), LV_ALIGN_CENTER, POS_X_DAY, POS_Y_PLUS);
+  lv_obj_set_style_local_value_str(btnDayPlus, LV_BTN_PART_MAIN, LV_STATE_DEFAULT, "+");
+  lv_obj_set_event_cb(btnDayPlus, event_handler);
+
+  btnDayMinus = lv_btn_create(lv_scr_act(), NULL);
+  btnDayMinus->user_data = this;
+  lv_obj_set_size(btnDayMinus, 50, 40);
+  lv_obj_align(btnDayMinus, lv_scr_act(), LV_ALIGN_CENTER, POS_X_DAY, POS_Y_MINUS);
+  lv_obj_set_style_local_value_str(btnDayMinus, LV_BTN_PART_MAIN, LV_STATE_DEFAULT, "-");
+  lv_obj_set_event_cb(btnDayMinus, event_handler);
+
+  btnMonthPlus = lv_btn_create(lv_scr_act(), NULL);
+  btnMonthPlus->user_data = this;
+  lv_obj_set_size(btnMonthPlus, 50, 40);
+  lv_obj_align(btnMonthPlus, lv_scr_act(), LV_ALIGN_CENTER, POS_X_MONTH, POS_Y_PLUS);
+  lv_obj_set_style_local_value_str(btnMonthPlus, LV_BTN_PART_MAIN, LV_STATE_DEFAULT, "+");
+  lv_obj_set_event_cb(btnMonthPlus, event_handler);
+
+  btnMonthMinus = lv_btn_create(lv_scr_act(), NULL);
+  btnMonthMinus->user_data = this;
+  lv_obj_set_size(btnMonthMinus, 50, 40);
+  lv_obj_align(btnMonthMinus, lv_scr_act(), LV_ALIGN_CENTER, POS_X_MONTH, POS_Y_MINUS);
+  lv_obj_set_style_local_value_str(btnMonthMinus, LV_BTN_PART_MAIN, LV_STATE_DEFAULT, "-");
+  lv_obj_set_event_cb(btnMonthMinus, event_handler);
+
+  btnYearPlus = lv_btn_create(lv_scr_act(), NULL);
+  btnYearPlus->user_data = this;
+  lv_obj_set_size(btnYearPlus, 50, 40);
+  lv_obj_align(btnYearPlus, lv_scr_act(), LV_ALIGN_CENTER, POS_X_YEAR, POS_Y_PLUS);
+  lv_obj_set_style_local_value_str(btnYearPlus, LV_BTN_PART_MAIN, LV_STATE_DEFAULT, "+");
+  lv_obj_set_event_cb(btnYearPlus, event_handler);
+
+  btnYearMinus = lv_btn_create(lv_scr_act(), NULL);
+  btnYearMinus->user_data = this;
+  lv_obj_set_size(btnYearMinus, 50, 40);
+  lv_obj_align(btnYearMinus, lv_scr_act(), LV_ALIGN_CENTER, POS_X_YEAR, POS_Y_MINUS);
+  lv_obj_set_style_local_value_str(btnYearMinus, LV_BTN_PART_MAIN, LV_STATE_DEFAULT, "-");
+  lv_obj_set_event_cb(btnYearMinus, event_handler);
+
+  btnSetTime = lv_btn_create(lv_scr_act(), NULL);
+  btnSetTime->user_data = this;
+  lv_obj_set_size(btnSetTime, 70, 40);
+  lv_obj_align(btnSetTime, lv_scr_act(), LV_ALIGN_CENTER, 0, 90);
```

Comment:
Use LV_ALIGN_IN_BOTTOM_MID instead. Make the button wider (120px?) and taller if possible to make it easier to press.

Reply:
Done.